Vagrantfile

```diff
@@ -77,6 +78,19 @@ def mount_folders(config, mount_options)
 end
 
 Vagrant.configure(2) do |config|
+  # Optional plugin for reading from env file
+  if Vagrant.has_plugin?('vagrant-env')
+    config.env.enable
+  end
+
+  # Default box
+  config.vm.box = base_boxes.default
+
+  # Custom box
+  if ENV.has_key?('VAGRANT_BOX')
```

Comment:
Please also update the comment at the top of this page to mention `VAGRANT_BOX` and how one could use it.